internal/helm/helm.go

```diff
@@ -0,0 +1,102 @@
+package helm
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/kyma-incubator/kymactl/internal/net"
+	"github.com/pkg/errors"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
+	"k8s.io/helm/pkg/helm"
+	"k8s.io/helm/pkg/helm/environment"
+	"net/http"
+	"net/url"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+type Client struct {
+	helm.Interface
+	forwarder chan<- struct{}
+}
+
+func New(settings *environment.EnvSettings, config *rest.Config) (*Client, error) {
+	forwarder, err := setupTillerConnection(settings, config)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Somehow the helm client makes a lot of verbose logs now, can you please double check

Reply:
Can you paste an example? I haven't noticed.

Reply:
I had an env variable active "DEBUG=true". After removing that the logs disappeared.